### PR TITLE
DP WG annual report 2021

### PR DESCRIPTION
**Pull request by xing-yang**
**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

**Review by phuongatemc — APPROVED** [review]
Look OK to me.

File: wg-data-protection/annual-report-2021.md [hunk 1]
```diff
@@ -1,62 +1,76 @@
...
+- Kubernetes Data Protection with Velero: https://docs.google.com/document/d/15tLCV3csvjHbKb16DVk-mfUmFry_Rlwo-2uG6KNGsfw/edit#heading=h.iekhl8nl58lo
 
 ## Project health
 
 1. What's the current roadmap until completion of the working group?
 
-   -
-   -
-   -
+We have identified the missing building blocks for supporting Data Protection in Kubernetes in our white paper: https://github.com/kubernetes/community/blob/master/wg-data-protection/data-protection-workflows-white-paper.md#what-are-the-missing-building-blocks-in-kubernetes.  We will continue to work on them until all the missing pieces are available in Kubernetes.
```
Comment (parispittman) [inline wg-data-protection/annual-report-2021.md:35]: for the rest of the work that needs to be done, what sig(s) would be owning the artifacts?

File: wg-data-protection/annual-report-2021.md [hunk 2]
```diff
@@ -1,62 +1,76 @@
...
 
 2. Does the group have contributors from multiple companies/affiliations?
 
-   -
+Yes
```
Comment (parispittman) [inline wg-data-protection/annual-report-2021.md:39]: can you list a few of the top contributing orgs here?

**Review by parispittman — CHANGES_REQUESTED** [review]

File: wg-data-protection/annual-report-2021.md (hunk 2 above)
Comment (shuguet) [reply wg-data-protection/annual-report-2021.md:39]: Based on the meeting minutes log here: https://docs.google.com/document/d/15tLCV3csvjHbKb16DVk-mfUmFry_Rlwo-2uG6KNGsfw/edit#

Those are the companies currently listed as having shown interest in this WG: Arrikto, Catalogic Software, Cohesity, Commvault, Dell EMC, Druva, Google, HPE, IBM, Infinidat, Kasten by Veeam, LINBIT, Linkedin, MayaData, Microsoft, Mongo, NetApp, Pure Storage (PortWorx), Red Hat, Rubrik, SUSE, Trilio, Veritas, VMware

File: wg-data-protection/annual-report-2021.md (hunk 1 above)
Comment (xing-yang) [reply wg-data-protection/annual-report-2021.md]: Updated with sig(s) info.

File: wg-data-protection/annual-report-2021.md (hunk 2 above)
Comment (xing-yang) [reply wg-data-protection/annual-report-2021.md]: Referenced the supporting companies in the agenda doc. Also listed companies actively participating in the WG.

**Comment by parispittman** [comment]
/lgtm

thank you!!

**Pull request merged**